Soft Allocation Journey.md

```diff
@@ -0,0 +1,193 @@
+# Introduction to Soft Allocation Brokering
+
+Hotwax Commerce's Soft Allocation Brokering feature performs real-time eligibility checks for same-day delivery of items in a customer's shopping cart. By evaluating the list of items in the cart along with the customer's zip code, Soft Allocation Brokering ensures compliance with predefined criteria for same-day shipping. 
+
+This process guarantees a seamless and efficient validation of same-day shipping eligibility, aligning with the requirements of various shipping carriers.
+
+Internally, the Soft Allocation process adheres to a structured set of steps: 
+
+
+### Step 1: Obtain the customer's latitude and longitude
+
+To check eligible same-day shipping facilities near the customer, the latitude and longitude coordinates of the customer are required. There are two ways to obtain these coordinates:
+
+a. If the customer inputs their postal code,
+  - Use the `postcodeLookup` API to convert the postal code into latitude and longitude coordinates.
```

Comment:
We should ad a link here to post code lookup api where readers can see more details on how to get lat long from zipcode